### PR TITLE
Add post-stop plugn trigger

### PR DESCRIPTION
**Pull request by kdomanski**
This will enable things like marking applications as manually stopped and preventing them from being restarted after reboot.

### CODE REVIEW

File: plugins/ps/commands [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-[[ " ps ps:start ps:stop ps:rebuild ps:rebuildall ps:restart ps:restartall ps:scale help ps:help " == *" $1 "* ]] || exit $DOKKU_NOT_IMPLEMENTED_EXIT
+[[ " ps ps:start ps:stop ps:rebuild ps:rebuildall ps:restart ps:restartall ps:restore ps:scale help ps:help " == *" $1 "* ]] || exit $DOKKU_NOT_IMPLEMENTED_EXIT
```
Comment (josegonzalez) [inline plugins/ps/commands]: Why this change?

Comment (kdomanski) [inline plugins/ps/commands]: Ooops, mixed up from some other work. Removing.
